lib/csetgrp.gi

```diff
@@ -789,6 +789,43 @@ local a,r;
 end);
 
 
+InstallMethod(Intersection2, "general groups", IsIdenticalObj,
```

Comment:
minor comment, change this to "general cosets"

Reply:
Thanks, done.